frontend/src/metabase-lib/types.ts

```diff
@@ -229,8 +246,8 @@ export type DrillThruDisplayInfo =
   | UnderlyingRecordsDrillThruInfo;
 
 export interface Dimension {
-  column: Record<string, unknown>;
+  column: DatasetColumn;
   value?: any;
 }
 
-export type DataRow = Array<{ col: Record<string, unknown>; value: any }>;
+export type DataRow = Array<{ col: DatasetColumn; value: any }>;
```

Comment:
Should `value` be of a [`RowValue` type](https://github.com/metabase/metabase/blob/bb00748b09aeb9fa219d08ea5bcff058ab223756/frontend/src/metabase-types/api/dataset.ts#L10)?